src/command_modules/azure-cli-documentdb/azure/cli/command_modules/documentdb/custom.py

```diff
@@ -23,22 +23,24 @@
 def cli_documentdb_create(client,
                           resource_group_name,
                           name,
-                          resource_group_location,
                           locations,
                           default_consistency_level=None,
                           max_staleness_prefix=100,
                           max_interval_in_seconds=5,
                           ip_range_filter=None):
     # pylint:disable=line-too-long
-    """Create new DocumentDB Database Account
-    :param resource_group_name: Name of resource group
-    :param name: Name of DocumentDB Database Account
-    :param location: Resource group location
-    """
+
     consistency_policy = None
-    if not default_consistency_level is None:
+    if default_consistency_level is not None:
         consistency_policy = ConsistencyPolicy(default_consistency_level, max_staleness_prefix, max_interval_in_seconds)
```

Comment:
Also, for clarity in your help text, for these three parameters, you could add the `arg_group='Consistency Policy'` kwarg to their `register_cli_argument` lines. This would cause these parameters to be visually grouped in help. Up to you whether you think that would enhance clarity or not, but wanted to point out the option.